.github/workflows/release.yml

```diff
@@ -6,8 +6,21 @@ on:
       - 'v*.*.*'
 
 jobs:
-  build-linux:
-    runs-on: ubuntu-latest
+  build:
+    strategy:
+      matrix:
+        rust-toolchain: [stable]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        arch: [amd64, arm64]
+        include:
+          - os: ubuntu-latest
+            name: linux
+          - os: macos-latest
+            name: darwin
+            rust-target: x86_64-apple-darwin
```

Comment:
I _think_ `${{ matrix.rust-target }}` will be an empty string for the other OS' and so I'm presuming that's not a problem when it comes to the `actions-rs/toolchain@v1` step's `target` property, which was only set for macOS.